sql/2021/http/percentiles_of_resources_loaded_over_HTTP2_or_better_per_site.sql

```diff
@@ -0,0 +1,25 @@
+# standardSQL
+# Percentiles of sites that load resources of HTTP/2 or above
+SELECT
+  client,
+  percentile,
+  APPROX_QUANTILES(http2_3_pct, 1000)[OFFSET(percentile * 10)] AS http2_or_above
+FROM (
+  SELECT
+    client,
+    page,
+    COUNTIF(LOWER(protocol) IN ('http/2', 'http/3', 'quic', 'h3-29', 'h3-q050')) / COUNT(0) AS http2_3_pct
+  FROM
+    `httparchive.almanac.requests`
+  WHERE
+    date = '2021-07-01'
+  GROUP BY
+    client,
+    page),
+  UNNEST([5, 6, 7, 8, 9, 10, 25, 50, 75, 90, 95, 100]) AS percentile
```

Comment:
What's the rationale for these unusual percentiles?

Reply:
Straight copy from last year. Will encourage to only use the "standard ones" if we do end up using this (at which point I might update the query to avoid this conversation next year!). For now I'd like to leave it the same to allow like for like comparison.